docs/user/tutorial.rst

```diff
@@ -27,9 +26,47 @@ Now run the ``mnist.py`` example:
 
   python mnist.py
 
-XXX Explain output
+You will get the following output:
+
+.. code-block::text
+
+  Loading data...
+  Building model and compiling functions...
+/usr/local/lib/python2.7/dist-packages/Lasagne-0.1dev-py2.7.egg/lasagne/layers/helper.py:52: UserWarning: get_all_layers() has been changed to return layers in topological order. The former implementation is still available as get_all_layers_old(), but will be removed before the first release of Lasagne. To ignore this warning, use `  warnings.filterwarnings('ignore', '.*topo.*')`.
+    warnings.warn("get_all_layers() has been changed to return layers in "
```

Comment:
I think this warning shouldn't be replicated in the tutorial, as it will go away in about 3-4 weeks anyway.